src/main/java/net/trollyloki/murdermystery/game/Game.java

```diff
@@ -41,6 +44,8 @@
     // Stores the UUID of the player that the potato will kill eventually - this will change several times!
     private UUID potatoVictim = null;
     private ArmorStand droppedBow;
+    private boolean potatoCooldown = false;
+    private Firework firework = null;
```

Comment:
No point in storing this as a field since you can only use the entity once anyway